sql/_project_record_ids.sql

```diff
@@ -34,19 +34,22 @@ project_record_join AS (
 	FROM dbscan a
 ),
 all_intersections AS (
-	SELECT 
+	SELECT
 		ST_UNION(ST_INTERSECTION(a.geom, b.geom)) as intersect_geom
 	FROM  project_record_join a, project_record_join b
 	WHERE a.record_id < b.record_id
 	AND a.records_in_project > 1
 	AND b.records_in_project > 1
 	AND a.no_geom = 0
 	AND b.no_geom = 0
+	AND a.id = b.id
+	GROUP BY a.id
 )
 SELECT
 	array_agg(a.record_id) as project_record_ids
-INTO _project_record_ids
 FROM project_record_join a, all_intersections b
-WHERE ST_OVERLAPS(a.geom, b.intersect_geom)
+WHERE (ST_Overlaps(a.geom, b.intersect_geom) or ST_Contains(a.geom, b.intersect_geom))
```

Comment:
this is nitpicky but the "or" should be uppercase to follow the convention in this SQL file